extensions/url-tools/src/components/ActionListSection.tsx

```diff
@@ -0,0 +1,27 @@
+import { List, ActionPanel, Action } from "@raycast/api";
+
+export const ActionListSection = ({
+  title,
+  text,
+  needFormat = false,
+}: {
+  title: string;
+  text: string;
+  needFormat?: boolean;
+}) => {
+  return (
+    <List.Section title={title}>
+      <List.Item
+        title={text}
+        actions={
+          <ActionPanel>
+            <Action.CopyToClipboard
+              content={needFormat ? JSON.stringify(JSON.parse(text), null, 2) : text}
+              shortcut={{ modifiers: ["ctrl"], key: "c" }}
```

Comment:
Could you change this to another shortcut since its interfering with the search bar

Reply:
Got it. Delete the shortcut is also fine.